<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::GoHome"/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <widget class="QStackedWidget" name="pagelayoutwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="mainmenu_page">
    <widget class="QPushButton" name="manualcontrol_page_btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual Control</string>
     </property>
    </widget>
    <widget class="QPushButton" name="systeminfo_page_btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>System Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editsettings_page_btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Settings</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="monitordata_page_btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>320</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Monitor Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="mainmenu_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Main Menu</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber {

background-color: darkgreen;

}</string>
     </property>
     <property name="value" stdset="0">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="systeminfo_page">
    <widget class="QLabel" name="systeminfo_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>202</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>System Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="systeminfo_back_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="monitordata_page">
    <widget class="QLabel" name="monitordata_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>202</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Monitor Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="monitordata_back_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="editsettings_page">
    <widget class="QPushButton" name="irrigation_page_btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Irrigation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editsettings_back_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="irrigation_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="manualcontrol_page">
    <widget class="QPushButton" name="manualcontrol_back_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="manualcontrol_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual Control</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="irrigation_page">
    <widget class="QPushButton" name="humidsetpoint_btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Humid Setpoint</string>
     </property>
    </widget>
    <widget class="QPushButton" name="irrigation_back_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="irrigation_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Irrigation</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {

background-color: grey;

}</string>
   </property>
   <widget class="QLCDNumber" name="lcdNumbertime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber {

background-color: darkgreen;

}</string>
    </property>
    <property name="value" stdset="0">
     <double>5.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
